src/test/java/it/pagopa/interop/probing/caller/unit/util/ClientUtilTest.java

```diff
@@ -0,0 +1,163 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.core.io.Resource;
+import org.springframework.util.StreamUtils;
+import feign.Request;
+import feign.RequestTemplate;
+import feign.Response;
+import io.awspring.cloud.messaging.listener.SimpleMessageListenerContainer;
+import it.pagopa.interop.probing.caller.client.FeignRestClient;
+import it.pagopa.interop.probing.caller.client.FeignSoapClient;
+import it.pagopa.interop.probing.caller.config.client.RestClientConfig;
+import it.pagopa.interop.probing.caller.config.client.SoapClientConfig;
+import it.pagopa.interop.probing.caller.dto.impl.EserviceContentDto;
+import it.pagopa.interop.probing.caller.dto.impl.TelemetryDto;
+import it.pagopa.interop.probing.caller.soap.probing.ProbingRequest;
+import it.pagopa.interop.probing.caller.soap.probing.ProbingResponse;
+import it.pagopa.interop.probing.caller.util.ClientUtil;
+import it.pagopa.interop.probing.caller.util.EserviceStatus;
+import it.pagopa.interop.probing.caller.util.EserviceTechnology;
+import it.pagopa.interop.probing.caller.util.logging.Logger;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+class ClientUtilTest {
+
+  @Mock
+  RestClientConfig restClientConfig;
+
+  @Mock
+  SoapClientConfig soapClientConfig;
+
+  @Mock
+  FeignRestClient feignRestClient;
+
+  @Mock
+  FeignSoapClient feignSoapClient;
+
+  @Mock
+  private Logger logger;
+
+  @MockBean
+  private SimpleMessageListenerContainer simpleMessageListenerContainer;
+
+  @InjectMocks
+  ClientUtil clientUtil;
+
+  @Value("classpath:data/bodyContent.json")
+  private Resource bodyContent;
+
+  private EserviceContentDto eserviceContentRestDto;
+  private EserviceContentDto eserviceContentSoapDto;
+  private TelemetryDto telemetryDto;
+  private TelemetryDto telemetryDtoKO;
+
+  @BeforeEach
+  void setup() {
+
+    String[] basePath = {"basePath1", "basePath2"};
+    eserviceContentRestDto = EserviceContentDto.builder().eserviceRecordId(1L)
+        .technology(EserviceTechnology.REST).basePath(basePath).build();
+    eserviceContentSoapDto = EserviceContentDto.builder().eserviceRecordId(1L)
+        .technology(EserviceTechnology.SOAP).basePath(basePath).build();
+    telemetryDto = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.OK)
+        .responseTime(12345L).koReason(null).checkTime("12345").build();
+    telemetryDtoKO = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.KO)
+        .koReason("KO").checkTime("12345").build();
+  }
+
+  @Test
+  @DisplayName("Test call probing - rest")
```

Comment:
shouldn't be there a `Test OK` and `Test KO` also for REST?

Reply:
done